modules/dqt/ajax/saveQuery.php

```diff
@@ -69,7 +69,7 @@
                                     ': ' .
                                     $_REQUEST['QueryName'];
 }
-$fields = json_decode($_REQUEST['Fields']);
+$fields = json_decode(strval($_REQUEST['Fields']));
```

Comment:
Hi @kongtiaowang what's the reason for this change? Also @ridz1208 do you remember a discussion about whether it's good to take the string value here or not because I somewhat remember us discussing whether bad or good.

Reply:
"$_REQUEST['Fields']" returns a object ({}),  we need a ("{}") to put inside json decode. that is the reason. 
Json_decode only takes string value inside the function.  
json_decode(
    **string** $json,
    ?bool $associative = null,
    int $depth = 512,
    int $flags = 0
): [mixed](https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.mixed)